docs/quick_start.md

```diff
@@ -22,7 +22,7 @@ The steps below will guide you through using model analyzer to analyze a simple
 
 1. Clone the repository and build the docker:
 ```
-$ git clone https://github.com/triton-inference-server/model_analyzer.git
+$ git clone https://github.com/triton-inference-server/model_analyzer.git -b r22.04
```

Comment:
Do we want to encourage people to use release branch instead of using main?  I think for most cases main will work fine.

Reply:
If there is a feature that MA relies on from server or Perf Analyzer that is not available in the released SDK/Server container, it will break the quick start. We want to make quick start guide as stable as possible since this is probably the first time that they are using Model Analyzer.

I have improved the instructions on using main branch in the docs/install.md for those who want to build MA on main.

Reply:
Yeah that makes sense.  @mc-nv are you ok updating this every release?